mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -739,6 +739,72 @@ class ItoFCastOpConversion : public OpConversionPattern<CastOp> {
   }
 };
 
+class TruncFConversion : public OpConversionPattern<arith::TruncFOp> {
+public:
+  TruncFConversion(const TypeConverter &typeConverter, MLIRContext *context)
```

Comment:
Maybe you can use the `using OpConversionPattern ..` to pull the constructor from the base class since the current one is empty?

Reply:
Yes :)